packages/lesswrong/components/sequences/TopPostsPage.tsx

```diff
@@ -1,7 +1,7 @@
-import React, { useEffect, useRef, useState } from 'react';
+import React, { useEffect, useMemo, useRef, useState } from 'react';
```

Comment:
Unused import `useMemo` (allow, as per my other comment, you might want to actually use it rather than remove it).